app/assets/javascripts/components/form/form_body.es6.jsx

```diff
@@ -8,6 +15,13 @@ class FormBody extends Component {
     return {};
   }
 
+  updateValue(key, value) {
+    fields = {};
+    fields[key] = value;
+    this.setState(fields);
+    console.log(this.state);
```

Comment:
Can you leave this method `updateValue` empty for now and leave a `TODO(Sonia): Finish this method definition.` comment inside of it instead?